includes/class-wc-payments.php

```diff
@@ -561,6 +559,16 @@ public static function init() {
 
 		self::maybe_display_express_checkout_buttons();
 
+		// Insert the Stripe Payment Messaging Element only if there is at least one BNPL method enabled.
+		$activated_bnpl_payment_methods = array_intersect(
+			Payment_Method::BNPL_METHODS,
+			self::get_gateway()->get_upe_enabled_payment_method_ids()
+		);
+
+		if ( ! empty( $activated_bnpl_payment_methods ) ) {
```

Comment:
`if ( [] !== $activated_bnpl_payment_methods )` would be a better fit for out strict types conventions.

Reply:
Thanks for the early review 🙌 

I have removed this here - 8275876